code/frameworks/sveltekit/src/preview.ts

```diff
@@ -136,3 +148,11 @@ const svelteKitMocksDecorator: Decorator = (Story, ctx) => {
 };
 
 export const decorators: Decorator[] = [svelteKitMocksDecorator];
+
+export const beforeEach: Preview['beforeEach'] = async (ctx) => {
+  const svelteKitParameters: SvelteKitParameters = ctx.parameters?.sveltekit_experimental ?? {};
+
+  setAppStatePage(svelteKitParameters?.state?.page);
+  setAppStateNavigating(svelteKitParameters?.state?.navigating);
+  setAppStateUpdated(svelteKitParameters?.state?.updated);
+};
```

Comment:
@xeho91 so presumably the failing tests indicate that we need to move this into the decorator instead?

Reply:
Not entirely sure bit I think no. I'm puzzled by this error. The mocks for `$app/state` doesn't use `getContext` at all.

The mocks for `$app/stores` do use the `getContext`, and they remain in the decorator (I only improved for set functions).